sieves/tasks/predictive/ner/bridges.py

```diff
@@ -0,0 +1,524 @@
+            prompt_signature_desc=prompt_signature_desc,
+        )
+
+    @property
+    def _prompt_template(self) -> str | None:
+        return None
+
+    @property
+    def _prompt_signature_description(self) -> str | None:
+        return None
+
+    @property
+    def prompt_signature(self) -> list[str]:
+        return self._entities
+
+    @property
+    def inference_mode(self) -> glix_.InferenceMode:
+        return glix_.InferenceMode.ner
+
+    def integrate(self, results: Iterable[glix_.Result], docs: Iterable[Doc]) -> Iterable[Doc]:
+        docs_list = list(docs)
+        results_list = list(results)
+
+        class Entity(pydantic.BaseModel):
+            text: str
+            start: int
+            end: int
+            entity_type: str
+
+        class Entities(pydantic.BaseModel):
+            text: str
+            entities: list[Entity] = []
+
+        for doc, result in zip(docs_list, results_list):
+            entities_list = []
+            # Get the original text from the document
+            doc_text = doc.text if doc.text is not None else ""
+
+            if result:
+                for entity in result:
+                    if isinstance(entity, dict) and "text" in entity and "label" in entity:
+                        # Convert types safely
+                        try:
+                            entity_text = str(entity["text"]) if entity["text"] is not None else ""
+                            entity_start = int(entity["start"]) if entity["start"] is not None else 0
+                            entity_end = int(entity["end"]) if entity["end"] is not None else 0
+                            entity_type = str(entity["label"]) if entity["label"] is not None else ""
+
+                            entities_list.append(
+                                Entity(
+                                    text=entity_text,
+                                    start=entity_start,
+                                    end=entity_end,
+                                    entity_type=entity_type,
+                                )
+                            )
+                        except (ValueError, TypeError):
+                            # Skip entities that can't be properly converted
+                            continue
+
+            # Create the Entities object with the list of entities
+            entities_obj = Entities(text=doc_text, entities=entities_list)
+            doc.results[self._task_id] = entities_obj
+
+        return docs_list
+
+    def consolidate(
+        self, results: Iterable[glix_.Result], docs_offsets: list[tuple[int, int]]
+    ) -> Iterable[glix_.Result]:
+        results = list(results)
+
+        # Process each document (which may consist of multiple chunks)
+        for doc_offset in docs_offsets:
+            all_entities = []
+            char_offset = 0
+
+            # Process each chunk for this document
+            for chunk_idx in range(doc_offset[0], doc_offset[1]):
```

Comment:
Use the same nomenclature as other tasks are using when processing document chunks.